docs/templating_funcs.md

```diff
@@ -133,3 +133,11 @@ The `expandSecretRefs` function takes an object as the argument and expands ever
 ```yaml
 {{ $expandSecretRefs :=  $value | expandSecretRefs }}
 ```
+
+#### `include`
+The 'include' function allows including and rendering nested templates. The function returns the created template or an error if any occurred. It will load functions from `_*.tpl` files in the directory where the helmfile.yaml is located.
+
+For nested helmfile.yaml files, it will load `_*.tpl` files in the directory where each nested helmfile.yaml is located.
+```yaml
+{{ include "my-template" . }}
```

Comment:
The introduction here does not let users know how to use include. We can give an example. For example: For examples, please refer to test/integration/test-cases/include-template-func.